dask/dataframe/tests/test_indexing.py

```diff
@@ -549,3 +549,31 @@ def test_iloc_raises():
 
     with pytest.raises(IndexError):
         ddf.iloc[:, [5, 6]]
+
+
+@pytest.fixture
+def ddf():
```

Comment:
Let's just create this inline in the tests. A bit more repetition, but less indirection which I think is more valuable here.